.github/workflows/documentation.yml

```diff
@@ -36,6 +41,26 @@ jobs:
           command: doc
           args: --no-deps --workspace --features docsrs
 
+      - name: Build `matrix-sdk-crypto-nodejs` doc
+        run: |
+          cd bindings/matrix-sdk-crypto-nodejs
+          npm install
```

Comment:
Do we want to use `npm ci` instead in CI? I think that's generally prefered. Would probably affect other job parts too, so maybe sth. for a separate PR.

Reply:
would leave that to @Hywan to decide ...

Reply:
TIL `npm ci`. I also thought it was standing for continuous integration, but no, it's for clean install. How confusing right?

Each flow runs in a separate new clean fresh environment, so I don't think it will impact other jobs, but yeah, that's probably better anywhere, so that we are sure. Will address that in another PR.